applications/nrf5340_audio/doc/firmware_architecture.rst

```diff
@@ -19,6 +19,10 @@ The applications use the same code base, but use different :file:`main.c` files
 You might need to configure and program two applications for testing the interoperability, depending on your use case.
 See the testing steps for each of the application for more information.
 
+.. note::
+   From NCSDK 2.6.0 the nRF5340 Audio applications will build with the :ref:`ug_ble_controller_softdevice` by default.
+   The :ref:`_lib_bt_ll_acs_nrf53_readme` is deprecated and will be removed after NCSDK 2.7.0 release.
+
```

Comment:
```suggestion
.. note::
   From NCSDK 2.6.0 the nRF5340 Audio applications will build with the :ref:`ug_ble_controller_softdevice` by default.
   The :ref:`_lib_bt_ll_acs_nrf53_readme` is deprecated and will be removed at a later point in time.

```

Reply:
```suggestion
.. note::
   From the |NCS| v2.6.0, the nRF5340 Audio applications are by default built with the :ref:`ug_ble_controller_softdevice`.
   The :ref:`lib_bt_ll_acs_nrf53_readme` is :ref:`deprecated <api_deprecation>` and will be removed in one of the future releases.
```
Do we really need this note here? I think that with the note in the LE Audio controller docs, the change in the changelog, and the note in the QDID section, this will be enough to inform the users.

I'm writing this because this note is very much release-specific and not general as the rest of the page.

Reply:
I'm fine with removing it, so I'll do that. I saw that the deprecation is mentioned in the migration guide, so people affected by this should find the same information there